src/fraction.ts

```diff
@@ -34,20 +36,22 @@ export class Fraction {
   }
 
   inverted() {
-    return new Fraction(this.denominator, this.numerator);
+    if (this.isZero()) {
+      return new Fraction(0, 1);
```

Comment:
I think this should error no? The inverse of `0/1` is `1/0` which is not a valid fraction.

Reply:
The problem is that we might have some prices/volumes that are 0 and we would still like to be able to invert them. But you are right that this logic if wrong. I should find a way to express something like "infinity".

Reply:
Will you also do operations with 1/0? In case, what are the expectations?

Reply:
1/0 can happen if the price/volume was 0 for some reason and we invert it. In that case I feel it's fine to use the default JS behaviour, which evaluates to Infinity

Reply:
Wouldn't it be better to remove zero orders and keep fractions intuitive? Having to also think about infinities when checking fraction code could be difficult. For example, I'd expect 1/0+1/0 to return infinity, but I think currently it errors out. Is this fine with the current logic? Can some unreasonable operation ever happen, like zero times infinity?